### PR TITLE
Shellfish production area context & payload example

### PR DESCRIPTION
**Pull request by JDFleury**

### CODE REVIEW

File: jsonld-contexts/shellfish-production-areas.jsonld [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "@context": {
+    "ShellfishProductionArea": "https://smartdatamodels.org/ShellfishProductionArea"
```
Comment (bobeal) [inline jsonld-contexts/shellfish-production-areas.jsonld]: add it in `shellfish-farming.jsonld`?

Comment (JDFleury) [reply jsonld-contexts/shellfish-production-areas.jsonld]: I am not sure I understand. Do you want me to add the relation to shellfish-farming.jsonld for example like this `"ShellfishTable": "https://schema.org/ShellfishTable",` (That is a current example) ?

Comment (bobeal) [reply jsonld-contexts/shellfish-production-areas.jsonld]: yes, no need to create a new shellfish context, as there already exists one

Comment (bobeal) [reply jsonld-contexts/shellfish-production-areas.jsonld]: so just move this line to `shellfish-farming.jsonld`

Comment (JDFleury) [reply jsonld-contexts/shellfish-production-areas.jsonld]: Understood

File: jsonld-contexts/shellfish-farming.jsonld [hunk 2]
```diff
@@ -54,6 +54,7 @@
         "regulationText": "https://schema.org/regulationText",
         "sampleSize": "https://smart-data-models.org/sampleSize",
         "sanitaryStatus": "https://schema.org/sanitaryStatus",
+        "ShellfishProductionArea": "https://smart-data-models.org/ShellfishProductionArea",
```
Comment (bobeal) [inline jsonld-contexts/shellfish-farming.jsonld]: for all the new terms, use `vocab.egm.io` (instead of `smart-data-models.org`)

**Review by bobeal — APPROVED** [review]

**Comment by junjou** [comment]
Let's try to keep all the new directories and files in `camelCase`. If you all agree?

**Comment by JDFleury** [comment]
> Let's try to keep all the new directories and files in `camelCase`. If you all agree?

Yes I think that is best, was just following the current format. Would you like me to change the files to camelCase ?

**Review by junjou — APPROVED** [review]
OK

**Pull request merged**